web/pgadmin/static/js/sqleditor/geometry_viewer.js

```diff
@@ -326,7 +326,7 @@ function parseData(items, columns, columnIndex) {
 
   // generate map info content
   if (tooLargeDataSize || tooManyGeometries) {
-    infoList.push(supportedGeometries.length + ' of ' + items.length + ' geometries rendered.');
+    infoList.push(supportedGeometries.length + gettext(' of ') + items.length + gettext(' geometries rendered.'));
```

Comment:
This won't work with some languages. You can build the string as: `gettext("%d of %d geometries rendered." %(supportedGeometries.length, items.length))`